### ISSUE TITLE
A question about VQA part

### ISSUE THREAD

**Issue by maa04** [facebookresearch/EmbodiedQA#19]
```
if len(pos_queue) < 5:  
    pos_queue = train_loader.dataset.episode_pos_queue[len(
        pos_queue) - 5:] + pos_queue
```
In train_eqa.py, when the length of input frames of vqa_model less than 5, it will use episode_pos_queue[len(pos_queue)-5:], is there something "using standard pos" occur?

For example, when randomly put the agent in somewhere far away from target object, it can stop immediately, and get final 5 frames from standard pos_queue, which lead to a high accuracy.

How about replacing it with the following code?

`
                        pos_queue = [pos_queue[0].copy() for _ in range(5 - len(pos_queue))] + pos_queue
`
